api/test/test_ts_geojson.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+import datastore_pb2 as dstore
+from deepdiff import DeepDiff
+from formatters.geojson import convert_to_geojson
+from google.protobuf import json_format
+
+
+def actual_response_is_expected_response(actual_response, expected_path, **kwargs):
```

Comment:
I don't like the asymmetry of one coming in as a file name, and the other as a Python dict...